pic32/variants/Uno32/Board_Defs.h

```diff
@@ -155,6 +155,9 @@
 #define	PIN_INT3	8
 #define	PIN_INT4	35
 
+#define NOT_AN_INTERRUPT -1
+#define digitalPinToInterrupt(p) ((p) == PIN_INT0 ? 0 : ((p) == PIN_INT1 ? 1 : ((p) == PIN_INT2 ? 2 : ((p) == PIN_INT3 ? 3 : ((p) == PIN_INT4 ? 4 : NOT_AN_INTERRUPT)))))
```

Comment:
Is this macro actually used for anything? If it's not it should be removed (to avoid future confusion). If it is wanted for something in the future it will have to be implemented in all boards and also a default added to pins_arduino.h in the core.